src/components/Badge/__test__/Badge.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Badge } from "..";
+
+describe("Badge component", () => {
+    it("renders children", () => {
+        render(<Badge isBold={false}>Test</Badge>);
+        expect(screen.getByText("Test")).toBeInTheDocument();
+    });
+
+    it("renders left and right icons", () => {
+        const leftIcon = <span data-testid="left-icon">Left Icon</span>;
```

Comment:
Do you guys not use the `dt` prefix @yaswanth-deriv?
```suggestion
        const leftIcon = <span data-testid="left-icon">Left Icon</span>;
```

Reply:
We use dt but as it is test cases I used this format